src/Kernel/CompiledCode.class.st

```diff
@@ -928,8 +932,8 @@ CompiledCode >> withAllNestedLiteralsDo: aBlockClosure [
 	As a Block or Method can have literals in the nested blocks. 
 	This is used to query all the selectors used in a method for example"
 	
-	self withAllBlocks do: [ :aCompiledCode | 
-		aCompiledCode allLiterals do: aBlockClosure ]
+	self withAllBlocksDo: [ :aCompiledCode | 
+		aCompiledCode literals do: aBlockClosure ]
```

Comment:
We definitely do not need a class of a method to traverse here. But #literals do not look into pragmas (the last literal in general). 
I checked that users of this method are not affected by this change (they do not care about pragmas or whatever last literal can return)

Reply:
yes, correct!